### DISCUSSION TITLE
(Double) Log transformation of survival times

### DISCUSSION THREAD

**Discussion by zsewa** [sebp/scikit-survival#200] [Q&A] [answered]
Hi, thanks for your package. 

I have a question about the log transform of survival times suggested in the documentation IPYNB about the Regression Survival SVM. 

The IPYNB suggests transforming the survival time prior to training the SVM, e.g. calling the `fit` method. I have noticed the survival times are also log-transformed for regression objectives in the `fit` method of the SVM.

Do you suggest running this nested log transformation or is the advice in the documentation a relic?

Cheers!

**Comment by sebp** [comment, marked as answer]
@zsewa That's indeed a mistake in the documentation that should be fixed. Thanks for bringing this to my attention!

**Reply by zsewa** [reply]
Thanks for your quick reply :)
